### PR TITLE
samples project for demo and test resources

### PR DESCRIPTION
**Pull request by mitchdraft**
as we move docs to the gloo repo and include more of our tutorials in CI, it will be helpful to manage our examples/demo resources as a dedicated project.
I'm open to other organization strategies. This seems simple and appropriate enough, LMKWYT
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/1460

### CODE REVIEW

File: projects/examples/services/sleeper/main.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		sleepDuration, err = time.ParseDuration(sleepDurationSpec)
+		if err != nil {
+			if _, wErr := fmt.Fprintf(w, "could not parse duration request: %v", sleepDurationSpec); wErr != nil {
+				log.Printf("unable to write error message: %v", wErr)
+			}
+		}
+	}
+	time.Sleep(sleepDuration)
+	if _, err := fmt.Fprintf(w, "slept for %v", sleepDuration.String()); err != nil {
+		log.Printf("unable to respond with sleeep time: %v", err)
```
Comment (kdorosh) [inline projects/examples/services/sleeper/main.go]: nit: typo
```suggestion
		log.Printf("unable to respond with sleep time: %v", err)
```

File: projects/examples/services/sleeper/Makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+DOCKER_REPO ?= soloio
```
Comment (kdorosh) [inline projects/examples/services/sleeper/Makefile]: shouldn't this be `solo-io`?

File: projects/examples/services/sleeper/Makefile [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+APP_NAME := sleeper
+VERSION ?= v0.0.1
+LDFLAGS := ""
+GCFLAGS := all="-N -l"
+
+docker-push:
+	CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build -ldflags=$(LDFLAGS) -gcflags=$(GCFLAGS) -o $(APP_NAME)-linux-amd64 main.go
```
Comment (kdorosh) [inline projects/examples/services/sleeper/Makefile]: can we reuse the `GOARCH` and `GOOS` env vars here when naming the output binary?

File: projects/examples/services/sleeper/main.go [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+const (
+	envPort = "PORT"
+)
+
+func getEnvConfig() (*envConfig, error) {
+	ec := &envConfig{}
+	ec.port = os.Getenv(envPort)
+	if ec.port == "" {
+		return nil, fmt.Errorf("must specify a port with %v", envPort)
```
Comment (kdorosh) [inline projects/examples/services/sleeper/main.go]: perhaps:
```suggestion
		return nil, fmt.Errorf("must specify a port using the %v environment variable", envPort)
```

File: projects/examples/services/sleeper/main.go [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+		if err != nil {
+			if _, wErr := fmt.Fprintf(w, "could not parse duration request: %v", sleepDurationSpec); wErr != nil {
+				log.Printf("unable to write error message: %v", wErr)
+			}
+		}
+	}
+	time.Sleep(sleepDuration)
+	if _, err := fmt.Fprintf(w, "slept for %v", sleepDuration.String()); err != nil {
+		log.Printf("unable to respond with sleep time: %v", err)
+	}
```
Comment (kdorosh) [inline projects/examples/services/sleeper/main.go]: I think these `fmt.Fprintf`s should end with newlines for improved formatting and to ensure the console buffer is flushed

File: projects/examples/services/sleeper/manifest.yaml [hunk 6]
```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1beta1
```
Comment (kdorosh) [inline projects/examples/services/sleeper/manifest.yaml]: lets use a stable/k8s 1.16 compliant version of `Deployment`, https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md#deprecations-and-removals

File: projects/examples/services/sleeper/manifest.yaml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    matchLabels:
+      app: sleeper
+  template:
+    metadata:
+      labels:
+        app: sleeper
+    spec:
+      containers:
+        - name: sleeper
+          image: soloio/sleeper:v0.0.1
```
Comment (kdorosh) [inline projects/examples/services/sleeper/manifest.yaml]: do we want to rename the repo here to `solo-io`, to match our other images?

File: projects/examples/services/sleeper/Makefile (hunk 2 above)
Comment (mitchdraft) [reply projects/examples/services/sleeper/Makefile]: not for docker

Comment (mitchdraft) [reply projects/examples/services/sleeper/Makefile]: though I could put this in quay if you prefer

Comment (kdorosh) [reply projects/examples/services/sleeper/Makefile]: no strong preference on whether this lives in quay or dockerhub

**Comment by mitchdraft** [comment]
unrelated panic :/ in the tests, will make an issue
```
ests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": GATEWAY LOGS
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 2019/10/21 19:47:00 http: TLS handshake error from 10.76.1.1:56398: EOF
Step #15 - "regression-tests": {"level":"info","ts":1571687221.4498513,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:129","msg":"received validation request"}
Step #15 - "regression-tests": {"level":"info","ts":1571687221.450315,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:198","msg":"AdmissionReview for Kind=gateway.solo.io/v1, Kind=RouteTable, Namespace=kyxbhy8j Name= UID=8cd42cca-f43b-11e9-be33-42010a800046 patchOperation=DELETE UserInfo={system:serviceaccount:kube-system:namespace-controller 080a1607-badf-11e9-ba73-42010a800144 [system:serviceaccounts system:serviceaccounts:kube-system system:authenticated] map[]}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687221.4503815,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:184","msg":"Ready to write response ..."}
Step #15 - "regression-tests": {"level":"info","ts":1571687221.4504073,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:190","msg":"responded with review: {\"response\":{\"uid\":\"8cd42cca-f43b-11e9-be33-42010a800046\",\"allowed\":true}}"}
Step #15 - "regression-tests": 2019/10/21 19:47:02 http: TLS handshake error from 10.76.1.1:56422: EOF
Step #15 - "regression-tests": {"level":"info","ts":1571687224.0481992,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:129","msg":"received validation request"}
Step #15 - "regression-tests": {"level":"info","ts":1571687224.0487328,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:198","msg":"AdmissionReview for Kind=gateway.solo.io/v1, Kind=RouteTable, Namespace=xea78pur Name= UID=8e4a1938-f43b-11e9-be33-42010a800046 patchOperation=DELETE UserInfo={system:serviceaccount:kube-system:namespace-controller 080a1607-badf-11e9-ba73-42010a800144 [system:serviceaccounts system:serviceaccounts:kube-system system:authenticated] map[]}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687224.0488057,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:184","msg":"Ready to write response ..."}
Step #15 - "regression-tests": {"level":"info","ts":1571687224.0488298,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:190","msg":"responded with review: {\"response\":{\"uid\":\"8e4a1938-f43b-11e9-be33-42010a800046\",\"allowed\":true}}"}
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": GLOO LOGS
Step #15 - "regression-tests": 
Step #15 - "regression-tests": {"level":"info","ts":1571687222.0347064,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer","caller":"syncer/envoy_translator_syncer.go:121","msg":"Setting xDS Snapshot","key":"gateway-test-6519-1~gateway-proxy-v2","clusters":18,"listeners":1,"routes":1,"endpoints":18}
Step #15 - "regression-tests": {"level":"info","ts":1571687222.034826,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-gloo-9977 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687222.0349698,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-echo-app-for-robustness-test-5678 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687222.035036,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-gateway-443 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687222.0351186,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-gateway-proxy-v2-443 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687222.0351856,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-gloo-9988 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687222.0352345,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-testrunner-1234 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687222.035489,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-gloo-9966 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687222.0356379,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-gateway-proxy-v2-80 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687222.0357082,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer","caller":"syncer/envoy_translator_syncer.go:134","msg":"end sync 3426707809775826933"}
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": GATEWAY LOGS
Step #15 - "regression-tests": 
Step #15 - "regression-tests": {"level":"info","ts":1571687221.4498513,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:129","msg":"received validation request"}
Step #15 - "regression-tests": {"level":"info","ts":1571687221.450315,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:198","msg":"AdmissionReview for Kind=gateway.solo.io/v1, Kind=RouteTable, Namespace=kyxbhy8j Name= UID=8cd42cca-f43b-11e9-be33-42010a800046 patchOperation=DELETE UserInfo={system:serviceaccount:kube-system:namespace-controller 080a1607-badf-11e9-ba73-42010a800144 [system:serviceaccounts system:serviceaccounts:kube-system system:authenticated] map[]}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687221.4503815,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:184","msg":"Ready to write response ..."}
Step #15 - "regression-tests": {"level":"info","ts":1571687221.4504073,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:190","msg":"responded with review: {\"response\":{\"uid\":\"8cd42cca-f43b-11e9-be33-42010a800046\",\"allowed\":true}}"}
Step #15 - "regression-tests": 2019/10/21 19:47:02 http: TLS handshake error from 10.76.1.1:56422: EOF
Step #15 - "regression-tests": {"level":"info","ts":1571687224.0481992,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:129","msg":"received validation request"}
Step #15 - "regression-tests": {"level":"info","ts":1571687224.0487328,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:198","msg":"AdmissionReview for Kind=gateway.solo.io/v1, Kind=RouteTable, Namespace=xea78pur Name= UID=8e4a1938-f43b-11e9-be33-42010a800046 patchOperation=DELETE UserInfo={system:serviceaccount:kube-system:namespace-controller 080a1607-badf-11e9-ba73-42010a800144 [system:serviceaccounts system:serviceaccounts:kube-system system:authenticated] map[]}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687224.0488057,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:184","msg":"Ready to write response ..."}
Step #15 - "regression-tests": {"level":"info","ts":1571687224.0488298,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:190","msg":"responded with review: {\"response\":{\"uid\":\"8e4a1938-f43b-11e9-be33-42010a800046\",\"allowed\":true}}"}
Step #15 - "regression-tests": 2019/10/21 19:47:04 http: TLS handshake error from 10.76.1.1:56438: EOF
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": [AfterEach] with a mix of valid and invalid virtual services
Step #15 - "regression-tests":   /workspace/gopath/src/github.com/solo-io/gloo/test/kube2e/gateway/gateway_test.go:490
Step #15 - "regression-tests": [AfterEach] tests with virtual service
Step #15 - "regression-tests":   /workspace/gopath/src/github.com/solo-io/gloo/test/kube2e/gateway/gateway_test.go:168
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": GLOO LOGS
Step #15 - "regression-tests": 
Step #15 - "regression-tests": {"level":"info","ts":1571687241.6787245,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer","caller":"syncer/envoy_translator_syncer.go:121","msg":"Setting xDS Snapshot","key":"gateway-test-6519-1~gateway-proxy-v2","clusters":18,"listeners":1,"routes":1,"endpoints":18}
Step #15 - "regression-tests": {"level":"info","ts":1571687241.6789882,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-gloo-9966 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687241.6790488,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-gloo-9977 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687241.6792083,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-gloo-9988 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687241.679554,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-gateway-proxy-v2-443 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687241.6796198,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-testrunner-1234 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687241.6797268,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-echo-app-for-robustness-test-5678 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687241.67979,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-gateway-443 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687241.6798587,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer.reporter","caller":"reporter/reporter.go:174","msg":"wrote report {kube-svc:gateway-test-6519-1-gateway-proxy-v2-80 gateway-test-6519-1} : {Accepted  gloo map[] nil {} [] 0}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687241.6798842,"logger":"gloo.v1.event_loop.setup.v1.event_loop.envoyTranslatorSyncer","caller":"syncer/envoy_translator_syncer.go:134","msg":"end sync 18242223019762462087"}
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": GATEWAY LOGS
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 2019/10/21 19:47:16 http: TLS handshake error from 10.76.1.1:56794: EOF
Step #15 - "regression-tests": 2019/10/21 19:47:18 http: TLS handshake error from 10.76.1.1:56822: EOF
Step #15 - "regression-tests": {"level":"info","ts":1571687239.0494068,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:129","msg":"received validation request"}
Step #15 - "regression-tests": {"level":"info","ts":1571687239.0497963,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:198","msg":"AdmissionReview for Kind=gateway.solo.io/v1, Kind=VirtualService, Namespace=xea78pur Name= UID=97504b2d-f43b-11e9-be33-42010a800046 patchOperation=DELETE UserInfo={system:serviceaccount:kube-system:namespace-controller 080a1607-badf-11e9-ba73-42010a800144 [system:serviceaccounts system:serviceaccounts:kube-system system:authenticated] map[]}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687239.0499327,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:184","msg":"Ready to write response ..."}
Step #15 - "regression-tests": {"level":"info","ts":1571687239.0500033,"logger":"gateway.v1.event_loop.gateway.gateway-validation-webhook","caller":"k8sadmisssion/validating_admission_webhook.go:190","msg":"responded with review: {\"response\":{\"uid\":\"97504b2d-f43b-11e9-be33-42010a800046\",\"allowed\":true}}"}
Step #15 - "regression-tests": 2019/10/21 19:47:20 http: TLS handshake error from 10.76.1.1:56850: EOF
Step #15 - "regression-tests": {"level":"info","ts":1571687241.6154673,"logger":"gateway.v1.event_loop.gateway.v2.event_loop.translatorSyncer","caller":"syncer/translator_syncer.go:53","msg":"begin sync 14898899519335845897 (1 virtual services, 2 gateways, 0 route tables)"}
Step #15 - "regression-tests": {"level":"info","ts":1571687241.6166334,"logger":"gateway.v1.event_loop.gateway.v2.event_loop.translatorSyncer","caller":"syncer/translator_syncer.go:70","msg":"desired proxy {gateway-proxy-v2 gateway-test-6519-1}"}
Step #15 - "regression-tests": {"level":"info","ts":1571687241.6657107,"logger":"gateway.v1.event_loop.gateway.v2.event_loop.translatorSyncer","caller":"syncer/translator_syncer.go:88","msg":"end sync 14898899519335845897"}
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": •! Panic [78.135 seconds]
Step #15 - "regression-tests": Kube2e: gateway tests with virtual service with a mix of valid and invalid virtual services [It] preserves the valid virtual services in envoy when a virtual service has been made invalid 
Step #15 - "regression-tests": /workspace/gopath/src/github.com/solo-io/gloo/test/kube2e/gateway/gateway_test.go:519
Step #15 - "regression-tests": 
Step #15 - "regression-tests":   Test Panicked
Step #15 - "regression-tests":   runtime error: invalid memory address or nil pointer dereference
Step #15 - "regression-tests":   /usr/local/go/src/runtime/panic.go:82
Step #15 - "regression-tests": 
Step #15 - "regression-tests":   Full Stack Trace
Step #15 - "regression-tests":   	/usr/local/go/src/runtime/panic.go:522 +0x1b5
Step #15 - "regression-tests":   github.com/solo-io/gloo/test/kube2e/gateway_test.glob..func1.2.6.4()
Step #15 - "regression-tests":   	/workspace/gopath/src/github.com/solo-io/gloo/test/kube2e/gateway/gateway_test.go:533 +0x4d2
Step #15 - "regression-tests":   github.com/solo-io/gloo/vendor/github.com/onsi/ginkgo/internal/leafnodes.(*runner).runSync(0xc0000e2de0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
Step #15 - "regression-tests":   	/workspace/gopath/src/github.com/solo-io/gloo/test/kube2e/gateway/gateway_suite_test.go:47 +0x19b
Step #15 - "regression-tests":   testing.tRunner(0xc00005de00, 0x1f87ae0)
Step #15 - "regression-tests":   	/usr/local/go/src/testing/testing.go:865 +0xc0
Step #15 - "regression-tests":   created by testing.(*T).Run
Step #15 - "regression-tests":   	/usr/local/go/src/testing/testing.go:916 +0x357
Step #15 - "regression-tests":   
Step #15 - "regression-tests": ------------------------------
Step #15 - "regression-tests": SSSSSS[31m[1m"[0m[31mMon, 21 Oct 2019 19:47:29 UTC: github.com/solo-io/gloo/vendor/github.com/solo-io/go-utils/testutils/helper/install.go:198[0m[31m[1m"[0m	uninstalling gloo...
Step #15 - "regression-tests": 
Step #15 - "regression-tests": 
Step #15 - "regression-tests": Summarizing 1 Failure:
Step #15 - "regression-tests": 
Step #15 - "regression-tests": [Panic!] Kube2e: gateway tests with virtual service with a mix of valid and invalid virtual services [It] preserves the valid virtual services in envoy when a virtual service has been made invalid 
Step #15 - "regression-tests": /usr/local/go/src/runtime/panic.go:82
Step #15 - "regression-tests": 
Step #15 - "regression-tests": Ran 7 of 13 Specs in 760.733 seconds
Step #15 - "regression-tests": FAIL! -- 6 Passed | 1 Failed | 0 Pending | 6 Skipped
Step #15 - "regression-tests": --- FAIL: TestGateway (760.73s)
Step #15 - "regression-tests": FAIL
Step #15 - "regression-tests": 
Step #15 - "regression-tests": Ginkgo ran 1 suite in 13m26.667024514s
Step #15 - "regression-tests": Test Suite Failed
Finished Step #15 - "regression-tests"
```

https://storage.googleapis.com/solo-public-build-logs/logs.html?buildid=d6011b7c-ccc3-4bbf-94dd-146a5a34c3e2

/kick

**Pull request merged**